CIP-token-metadata-translations/README.md

```diff
@@ -0,0 +1,139 @@
+---
+CIP: ?
+Title: Extend token metadata for translations

```

Comment:
```suggestion
Title: CIP-25 - Extend token metadata for translations
```

might make sense to do something like this, as this is a strictly CIP25 extension